subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseJavaProjectIntegrationTest.groovy

```diff
@@ -42,6 +42,7 @@ class EclipseJavaProjectIntegrationTest extends AbstractEclipseIntegrationSpec {
         '1.6'   | 'JavaSE-1.6'
         '1.7'   | 'JavaSE-1.7'
         '1.8'   | 'JavaSE-1.8'
-        '1.9'   | 'JavaSE-1.9'
+        '1.9'   | 'JavaSE-9'
+        '1.10'  | 'JavaSE-10'
```

Comment:
I'm afraid there will be no Java 1.0, but 18.3 :) This will be a source of endless fun.